src/azure-cli/azure/cli/command_modules/storage/_validators.py

```diff
@@ -1307,3 +1295,49 @@ def validate_match_condition(namespace):
     if namespace.if_none_match:
         namespace = _if_none_match(if_none_match=namespace.if_none_match, **namespace)
         del namespace.if_none_match
+
+
+def validate_or_policy(namespace):
+    error_elements = []
+    if namespace.properties is None:
+        error_msg = "Please provide --policy in JSON format or the following arguments: "
+        if namespace.source_account is None:
+            error_elements.append("--source-account")
+
+        # Apply account name when there is no destination account provided
+        if namespace.destination_account is None:
+            namespace.destination_account = namespace.account_name
+
+        if error_elements:
+            error_msg += ", ".join(error_elements)
+            error_msg += " to initialize or Policy for storage account."
```

Comment:
>or [](start = 41, length = 2)

Use "Object Replication" instead of "or" in error message for more clearness.

Reply:
done